### PR TITLE
 [#1524] upgrade planet using nginx and the ui (fixes #1524)

### LINKED ISSUE #1524: Software Update or hack portainer
https://portainer.io/install.html

### PR DESCRIPTION
**Pull request by i5o**

### CODE REVIEW

File: docker/planet/upgrade.sh [hunk 1]
```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+echo "HTTP/1.0 200 OK"
+echo "Content-type: text/plain"
+echo ""
+curl --unix-socket /var/run/docker.sock -X POST "http://localhost/images/create?fromImage=treehouses/planet:db-init"
```
Comment (mappuji) [inline docker/planet/upgrade.sh]: Can we dry it perhaps? using function?

Comment (i5o) [reply docker/planet/upgrade.sh]: Sure. I will push this on wednesday

Comment (mappuji) [reply docker/planet/upgrade.sh]: cool!

File: docker/planet/upgrade.sh [hunk 2]
```diff
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
```
Comment (i5o) [inline docker/planet/upgrade.sh]: Bash is not installed on the docker image, thats why we are using sh

Comment (paulbert) [reply docker/planet/upgrade.sh]: Ok, will start a quick PR to install it.  `sh` doesn't support arrays, so throws an error.

**Pull request merged**